docker/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Dockerizing EIDA NG Mediator, Federator and StationLite images
+# Following the instructions from:
+#
+# https://github.com/EIDA/mediatorws
+#
+# Build Container:
+# docker build -t eida-federator:1.0 .
+#
+# Run Container:
+# docker run [-d] [--rm] [--name eida-federator] -p 8080:80 eidang_federator:1.0
+#
+# Modify running container:
+# docker exec -it eida-federator /bin/bash
+
+# Base image
+FROM ubuntu:18.04 
+
+# Add label metadata
+LABEL maintainer="Massimo Fares"
+LABEL email="massimo.fares@ingv.it"
+
+# System dependencies
+RUN apt-get update && apt-get install -y libxml2-dev \
+                                         libxslt-dev \
+                                         python-dev \
+                                         git \
+                                         build-essential \
+                                         python-setuptools \
+                                         zlib* \
+                                         python-pip \
+                                         pkg-config \
+                                         libpng-dev \
+                                         libfreetype6-dev \
+                                         libblas-dev \
+                                         liblapack-dev \
+                                         apache2 \
+                                         libapache2-mod-wsgi
+
+# Install the Python dependencies
+RUN pip install numpy obspy virtualenv
+
+# Get the source from GitHub:master
+RUN git clone https://github.com/EIDA/mediatorws.git /var/www/mediatorws
+
+# Create virtual environments for federator/stationlite
+RUN virtualenv /var/www/federator/venv && virtualenv /var/www/stationlite/venv
+
+# Install the services
+WORKDIR /var/www/mediatorws
+RUN make install
```

Comment:
Actually think I should be sourcing the venv during installation? And make install SERVICES="stationlite", SERVICE="federator" seperately?

Reply:
Yep. You seem to create `venv`s but you never use them. Within a container, however, IMO there is no need to use virtual environments.

So either you do:
```
RUN virtualenv /var/www/federator/venv
RUN . /var/www/federator/venv/bin/activate
RUN make install SERVICES=federator
RUN deactivate
```
(and the same for StationLite) or you install both services (i.e. `eida-federator` and `eida-stationlite`) into one `venv`:

```
RUN virtualenv /var/www/mediatorws/venv
RUN . /var/www/mediatorws/venv/bin/activate
RUN make install
RUN deactivate
```
or you install it globally.

Depending on using a virtual environment or not you have to adjust the configuration of the `WSGIDeamonProcess` directive within your `VirtualHost` configuration.).